descriptors/ModuleDescriptor-template.json

```diff
@@ -1127,6 +1127,31 @@
       "displayName": "Circulation - use circulation rules to get all matching notice policies",
       "description": "Apply circulation rules to get all matching notice policies"
     },
+    {
+      "permissionName": "circulation.rules.overdue-fine-policy-all.get",
+      "displayName": "Circulation - use circulation rules to get all matching overdue fine policies",
+      "description": "Apply circulation rules to get all matching overdue fine policies"
+    },
+    {
+      "permissionName": "circulation.rules.lost-item-all.get",
+      "displayName": "Circulation - use circulation rules to get all matching lost item policies",
+      "description": "Apply circulation rules to get all matching lost item policies"
+    },
+    {
+      "permissionName": "circulation.rules.overdue-fine-policy.get",
+      "displayName": "Circulation - use circulation rules to get all matching overdue fine policy",
```

Comment:
"all matching"? What's the difference between this and circulation.rules.overdue-fine-policy-all.get then?